ci-operator/jobs/openshift/release/openshift-release-release-4.4-periodics.yaml

```diff
@@ -512,13 +514,24 @@ periodics:
               RUN set -xeuo pipefail && \\
                   mkdir -p /srv/repo /tmp/working && \\
                   curl -L "${tar_url}" | tar xf - -C /srv/repo/ && \\
+                  cp /etc/yum.repos.d/fedora-updates-testing-modular.repo /tmp && \\
                   rm -rf /etc/yum.repos.d/* && \\
+                  cp /tmp/fedora-updates-testing-modular.repo /etc/yum.repos.d/ && \\
+                  sed -i 's/enabled=0/enabled=1/g' /etc/yum.repos.d/fedora-updates-testing-modular.repo && \\
                   rpm-ostree db list --repo /srv/repo "${ostree_ref}" > /tmp/packages && \\
+                  dnf module enable -y cri-o:1.16 && \\
+                  if ! grep -q cri-o /tmp/packages; then yumdownloader -y --disablerepo=* --enablerepo=updates-testing-modular --destdir=/tmp/rpms cri-o cri-tools; fi && \\
                   curl http://base-${BRANCH//\./-}-rhel8.ocp.svc > /etc/yum.repos.d/rhel8.repo && \\
-                  if ! grep -q cri-o /tmp/packages; then yumdownloader -y --disablerepo=* --enablerepo=rhel-8* --destdir=/tmp/rpms cri-o cri-tools; fi && \\
-                  if ! grep -q glusterfs-fuse /tmp/packages; then yumdownloader -y --disablerepo=* --enablerepo=rhel-8* --destdir=/tmp/rpms attr glusterfs glusterfs-client-xlators glusterfs-fuse glusterfs-libs psmisc; fi && \\
+                  if ! grep -q glusterfs-fuse /tmp/packages; then yumdownloader -y --disablerepo=* --enablerepo=rhel-8* --destdir=/tmp/rpms glusterfs glusterfs-client-xlators glusterfs-fuse glusterfs-libs psmisc; fi && \\
                   cd /tmp/working && \\
                   for i in \$(find /tmp/rpms/ -name *.rpm); do echo "Extracting \$i ..."; rpm2cpio \$i | cpio -div; done && \\
+                  cd usr/sbin && \
+                  ln -s xtables-nft-multi iptables && \\
```

Comment:
This looks weird.  Why is it necessary?

Reply:
By default Fedora 31 points to legacy iptables. This would switch alternatives to nft version (as in RHCOS). This however doesn't seem to help, so I'll keep it and use this PR to produce debug versions of OKD payload for now